src/test/scala/org/ergoplatform/nodeView/state/UtxoStateSpecification.scala

```diff
@@ -223,15 +223,15 @@ class UtxoStateSpecification extends ErgoPropertyTest with ErgoTransactionGenera
       // trying to apply transactions with missing data inputs
       val missedId: BoxId = ADKey @@ scorex.util.Random.randomBytes()
       us.boxById(missedId) shouldBe None
-      val missingDataInputs = (missedId +: existingBoxes).map(DataInput).toIndexedSeq
+      val missingDataInputs = (missedId +: existingBoxes).map(DataInput.apply).toIndexedSeq
```

Comment:
Is this necessary change?

Reply:
Not any more. Used to have `object DataInput`. Thanks for spotting this. Fixed.